backend/core/utils/time.py

```diff
@@ -9,3 +9,9 @@ def time_ago(**kwargs):
     """Return a time in the past as specified as keyword arguments."""
 
     return timezone.now() - timedelta(**kwargs)
+
+
+def time_ahead(**kwargs):
+    """Return a time in the future as specified as keyword arguments."""
```

Comment:
Perfect. Let's make the documentation more accurate.

```suggestion
    """
    Return a `datetime` in the future relative to now.

    The keyword arguments are directly passed to `datetime.timedelta`.
    """
```